libraries/typescript/Microsoft.Bot.Builder.Adapters.WeChat/src/accessTokenStorage.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @module wechat
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Storage, StoreItems } from 'botbuilder-core';
+import { WeChatAccessToken } from './weChatSchema';
+
+/**
+ * Storage provider for access token.
+ */
+export class AccessTokenStorage {
+    private Storage: Storage;
+
+    /**
+     * Creates an instance of access token storage.
+     * @param storage
+     */
+    constructor(storage: Storage) {
+        this.Storage = storage;
+    }
+
+    /**
+     * Saves store items to storage.
+     * @param key Item key to write to the storage.
+     * @param value Item value to write to the storage.
+     */
+    public async SaveAsync(key: string, value: WeChatAccessToken): Promise<void> {
```

Comment:
>SaveAsync [](start = 17, length = 9)

Your method names should all be camelCase.

Reply:
We also don't append "Async" in JavaScript. That's purely a C# pattern.

---
In reply to: [348231532](https://github.com/microsoft/BotFramework-WeChat/pull/3#discussion_r348231532) [](ancestors = 348231532)

Reply:
> We also don't append "Async" to method names in JavaScript. That's purely a C# pattern.
> 
> In reply to: [348231532](https://github.com/microsoft/BotFramework-WeChat/pull/3#discussion_r348231532) [](ancestors = 348231532)

I am referring this format from botbuilder-js repo. It has many methods like "public/private asnyc xxxx" format. It makes me confused. Should I remove all "async" in this repo?